smart_sa/templates/intervention/index.html

```diff
@@ -24,9 +24,13 @@
  		<p>
  		"Masivukeni" means "let's wake up" in English&mdash;so what are you waiting for? 
  		<br />
-		<a href="/practice/{{intervention.id}}/" id="start-link">Let's get started!</a>
+		<a href="/practice/{{intervention.id}}/" id="start-link">
+            <button id="btn-get-started"><h2>Use Masivukeni Online</h2></button>
```

Comment:
I don't recommend putting an `<h2>` tag inside a button - I dunno it just seems wrong. I would just remove the `<h2>` and change the font-size for this button in your CSS above.

Reply:
Yeah, its not semantic at all. I pushed some fixes.
Thanks!